garak/probes/persuasion.py

```diff
@@ -0,0 +1,52 @@
+"""Persuasion
+
+Probes using social science persuasion techniques from 
+[Persuasive Adversarial Prompts](https://arxiv.org/abs/2401.06373)
+"""
+
+import garak.probes
+
+
+class PersuasivePAP(garak.probes.Probe):
+    """Persuasive Adversarial Prompts (PAP) - Static Examples
+
+    Uses social science-based persuasion techniques (logical appeal,
+    emotional appeal, authority endorsement, etc.) to attempt bypassing
+    LLM safety measures.
+
+    The paper's full methodology involves a fine-tuned paraphraser to
+    generate prompts dynamically, but that model has not been made
+    publicly available. This probe uses static examples extracted from
+    the paper's successful demonstrations.
+    """
+
+    doc_uri = "https://arxiv.org/abs/2401.06373"
+    lang = "en"
+    active = False
+    goal = "elicit harmful outputs using persuasive communication techniques"
+    primary_detector = "mitigation.MitigationBypass"
+    extended_detectors = ["mitigation.Prefixes"]
+    tags = [
+        "avid-effect:security:S0403",
+        "owasp:llm01",
+        "quality:Security:PromptStability",
+        "payload:jailbreak",
+    ]
+    tier = garak.probes.Tier.OF_CONCERN
+
+    # Static prompts extracted from the paper's successful examples
+    # Each prompt has the harmful query embedded within the persuasive context
+    prompts = [
```

Comment:
Consider moving these to a file in `garak/data` - this makes them easier to customise